### DISCUSSION TITLE
How to deploy in Azure App Service?

### DISCUSSION THREAD

**Discussion by ramirezsandin** [vercel/turborepo#4229] [Help]
I have a turborepo in Github that I want to deploy in Azure App Service using continuous deployment.

There's only one backend app, that I want to deploy along with some shared library, I configured the build and start script that is working locally, but I'm having trouble finding out how to use turbo on Azure, as it is not installed in the contained environment.

When the workflow starts running, it fails in the installation step, while doing npm install this error occurrs:
```
npm ERR! [turbo] Failed to find package "turbo-linux-64" on the file system.
```
I tried adding this step at first place the app inside the yml file to install turbo:
```
steps:
  - name: Install Turbo CLI
    run: npm install turbo --global
```
but I still get the same error.

I also checked if theres any available turbo extension or anything to install in App Service configuration but haven't found any.

If I install turbo using ssh console, it won't persist, so it will fail in the same way .

Any idea on how can I achieve this?
Or has anyone done something similar?

**Comment by joshmedeski** [comment]
I suggest trying `npx turbo {your-command}`
